### ISSUE TITLE
Integrate with Travis CI

### ISSUE THREAD

**Issue by coleturner** [aria-51/aria-lights-out#14]
@JayKan what are your thoughts for the pipeline?

For starters I think we need Lint, maybe Prettier?

**Comment by JayKan** [comment]
@coleturner, do you wanna use `yarn` instead of `npm` on Travis? Because if we do proceed with `yarn`, we need to make sure we have node `>=8`, let me know

**Comment by coleturner** [comment]
Yeah we should use yarn everywhere, if it requires Node 8 that’s fine. 

There’s a great thread on Twitter about setting up the offline mirror for CI to speed up and prevent leaky exploits:

https://twitter.com/leeb/status/1017607265115750400?s=21

**Comment by JayKan** [comment]
Interesting cool, I'll take a look at this 👍 

Also, I'm having some trouble getting `@storybook/addon-storyshots` installed, any ideas on the below error:

<img width="1148" alt="screen shot 2018-07-15 at 11 18 57 pm" src="https://user-images.githubusercontent.com/1400300/42745047-7a44a058-8885-11e8-8cac-923555d56711.png">
